.github/workflows/deploy.yml

```diff
@@ -0,0 +1,40 @@
+# Basic workflow for Actions
+
+name: Build and Deploy to GitHub static pages
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the develop branch
+  push:
+    branches: [ develop ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1 # If you're using actions/checkout@v2 you must set persist-credentials to false in most cases for the deployment to work correctly.
+        with:
+          persist-credentials: false
+
+      - name: Install and Build 🔧 # This example project is built using npm and outputs the result to the 'build' folder. Replace with the commands required to build your project, or remove this step entirely if your site is pre-built.
+        run: |
+          npm install
+          npm run build
+          
+      - name: Deploy to Github Pages
+        uses: JamesIves/github-pages-deploy-action@3.7.1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH: develop # The branch the action should deploy to.
```

Comment:
Change to production

Reply:
🤔 But Bit web only has 2 branches, `Master` and `Develop`, @annabauza? 
<img width="815" alt="Screen Shot 2021-03-05 at 12 36 21 am" src="https://user-images.githubusercontent.com/29667122/109972424-6080b900-7d4b-11eb-9629-c19b5e82c53c.png">

Reply:
@mtreacy002 I agree I don't understand there isn't production branch.

Reply:
this is where the build folder is hosted. We need different branch to not override all your work @mtreacy002

Reply:
Ok, I will change

Reply:
@annabauza , based on what you've suggested [above](https://github.com/anitab-org/bridge-in-tech-web/pull/170#discussion_r589726715), can you please help create the production / staging branch for the repo? I don't seem to have access to that (can't find a way to add new branch to the repo). I think only admins can do that.

<img width="912" alt="Screen Shot 2021-03-13 at 1 25 12 pm" src="https://user-images.githubusercontent.com/29667122/111016010-98e55e80-83ff-11eb-91e9-b352612a20d7.png">